cfgov/prepaid_agreements/jinja2/prepaid_agreements/detail.html

```diff
@@ -14,7 +14,7 @@
 
     {%- import 'molecules/breadcrumbs.html' as breadcrumbs with context -%}
     {% set breadcrumb_items = [{
-        'href': '/data-research/prepaid-accounts/search-agreements/',
+        'href': breadcrumb,
```

Comment:
Minor thing, but maybe we could call this variable `detail_page_breadcrumb` to be more descriptive?

Reply:
Or, sorry, something like `search_page_url`, since that's what it links back to?

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5239/commits/9574f04596eea8b1ac2630af377e96387c8f5bf5